qa/L0_trace/opentelemetry_unittest.py

```diff
@@ -756,6 +757,22 @@ def test_sagemaker_invoke_trace_simple_model_context_propagation(self):
         time.sleep(5)
         self._test_simple_trace(headers=self.client_headers)
 
+    def test_trace_context_exposed_to_pbe(self):
```

Comment:
I think having a test for a case where `request.trace().get_context()` is `None` (such as server in `triton` mode) would be good.

Reply:
yeap, on the way

Reply:
added more tests: https://github.com/triton-inference-server/server/pull/6985/files#diff-27e9ca4cd8f292f0e5a94bbabb243e40fe0d47cd55164a0cb49ae6e093c9fbe7R1041-R1083